sdk/Javascript/src/MobileServiceClient.js

```diff
@@ -231,6 +231,7 @@ MobileServiceClient.prototype._request = function (method, uriFragment, content,
     if (!_.isNullOrEmpty["X-ZUMO-VERSION"]) {
         options.headers["X-ZUMO-VERSION"] = this.version;
     }
+    options.headers["ZUMO-API-VERSION"] = "2.0.0";
```

Comment:
That would be the SDK version, this is the new API version that is independent of the specific client bits, just what protocol it uses.  More details in the team one note.